app/include/user_modules.h

```diff
@@ -39,6 +39,7 @@
 //#define LUA_USE_MODULES_MDNS
 #define LUA_USE_MODULES_MQTT
 #define LUA_USE_MODULES_NET
+#define LUA_USE_MODULES_NET_INFO
```

Comment:
By default all new modules are disabled.

Reply:
Yep. This will be fixed. I'm afraid we are still far from merging this module...